packages/11ty/_plugins/transforms/outputs/epub/README.md

```diff
@@ -0,0 +1,3 @@
+# EPUB Transform
+
+The transform function is called for each template, with the output HTML and output path as arguments, at the end of the build step before the individual page content is written to file on disk. The EPUB transform intercepts the HTML content, creates a JSDOM instance, removes the base layout, wraps the body element in the layout from `transforms/outputs/epub/layout.js`, removes html- or pdf-only elements, and then writes the file to the output directory (`_site/epub/<sequence>_<filename>`).
```

Comment:
Edited for clarity though the description of the transform could also be a list.
```suggestion
The `epub` transform prepares the Eleventy build output for conversion to an EPUB by a separate library.

This transform is run at the end of the Eleventy build before individual page HTML files are written. For each template the [`epub/tranform`](transform.js) function is called with the HTML output from the build and the file output path.

The transform intercepts the HTML content and creates a `JSDOM` instance to replace the [`base` layout](_layouts/base.11ty.js), with the [`epub/layout`](transforms/outputs/epub/layout.js) and remove `html-only` or `pdf-only` elements before writing each file to the output directory with a sequence indentifier pattern: `_site/epub/<sequence>_<filename>`.

```